manifests/apache/redirect_vhost_https.pp

```diff
@@ -5,11 +5,14 @@
 define nebula::apache::redirect_vhost_https (
   Array[String] $serveraliases = [],
   String $target = "https://www.${title}/",
-  String $ssl_cn = $title,
+  $priority = false,
 ) {
-  nebula::apache::www_lib_vhost { "${title}-https":
-    ssl             => true,
-    ssl_cn          => $ssl_cn,
+  nebula::apache::redirect_vhost_http { "${title}":
+    serveraliases => $serveraliases
+  }
+  apache::vhost { "${title}-redirect-https":
```

Comment:
While in the _current_ functionality apache doesn't terminate ssl, the replacement should; therefore, I think you'll want to keep what was there before with the `$ssl_cn` parameter that gets passed through to the `nebula::apache::www_lib_vhost` rather than using `apache::vhost` here directly.